vnext/src-win/Libraries/Text/Text.windows.js

```diff
@@ -215,189 +225,98 @@ const Text: React.AbstractComponent<
+        verticalAlignToTextAlignVerticalMap[processedStyle.verticalAlign];
+      // $FlowFixMe[cannot-write]
+      delete processedStyle.verticalAlign;
+    }
   }
 
-  const _hasOnPressOrOnLongPress =
-    props.onPress != null || props.onLongPress != null;
+  const _nativeID = id ?? nativeID;
 
+  const hasTextAncestor = useContext(TextAncestor);
   if (hasTextAncestor) {
     return (
       <NativeVirtualText
         {...restProps}
         {...eventHandlersForText}
-        accessibilityLabel={ariaLabel ?? accessibilityLabel}
+        accessibilityLabel={_accessibilityLabel}
+        accessibilityState={_accessibilityState}
         accessibilityLevel={ariaLevel ?? accessibilityLevel} // Windows
         accessibilityPosInSet={ariaPosinset ?? accessibilityPosInSet} // Windows
         accessibilitySetSize={ariaSetsize ?? accessibilitySetSize} // Windows
+        isHighlighted={isHighlighted}
+        isPressable={isPressable}
+        nativeID={_nativeID}
+        numberOfLines={numberOfLines}
+        ref={forwardedRef}
+        selectable={_selectable}
+        selectionColor={selectionColor}
+        style={processedStyle}
+      />
+    );
+  }
+
+  // If the disabled prop and accessibilityState.disabled are out of sync but not both in
+  // falsy states we need to update the accessibilityState object to use the disabled prop.
+  if (
+    _disabled !== _accessibilityStateDisabled &&
+    ((_disabled != null && _disabled !== false) ||
+      (_accessibilityStateDisabled != null &&
+        _accessibilityStateDisabled !== false))
+  ) {
+    _accessibilityState = {..._accessibilityState, disabled: _disabled};
+  }
+
+  const _accessible = Platform.select({
+    ios: accessible !== false,
+    android:
+      accessible == null ? onPress != null || onLongPress != null : accessible,
+    default: accessible !== false,
+  });
+
+  return (
+    <TextAncestor.Provider value={true}>
+      <NativeText
+        {...restProps}
+        {...eventHandlersForText}
+        accessibilityLabel={_accessibilityLabel}
         accessibilityState={_accessibilityState}
+        accessible={_accessible}
+        accessibilityLevel={ariaLevel ?? accessibilityLevel} // Windows
+        accessibilityPosInSet={ariaPosinset ?? accessibilityPosInSet} // Windows
+        accessibilitySetSize={ariaSetsize ?? accessibilitySetSize} // Windows
         allowFontScaling={allowFontScaling !== false}
         disabled={_disabled}
         ellipsizeMode={ellipsizeMode ?? 'tail'}
         isHighlighted={isHighlighted}
+        nativeID={_nativeID}
         isPressable={isPressable}
-        nativeID={id ?? nativeID}
         numberOfLines={numberOfLines}
         ref={forwardedRef}
         selectable={_selectable}
         selectionColor={selectionColor}
-        style={style}
+        style={processedStyle}
       />
-    );
-  } else {
-    let styleProps: ViewStyleProp = (restProps.style: any);
-    if (
-      styleProps &&
```

Comment:
@acoates-ms this may be responsible for the change in borders for text? Not sure if this was intentional or a bad merge that I should fix.

Reply:
if you add this back in, does that restore the snapshot tests to the original?

Reply:
I think so - but I think we have an extra if/else block in the Text.windows.js file which was removed in the merge which should not have been. This extra block includes the `styleProps` variable, so adding it back in should resolve the issue.